lib/ansible/plugins/callback/telegraf.py

```diff
@@ -0,0 +1,290 @@
+# (C) 2017-2018, VMware, Inc. All Rights Reserved.  
+# SPDX-License-Identifier: GPL-3.0
+# (C) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: telegraf
+    callback_type: metrics
+    requirements:
+      - callback_whitelist = telegraf[,others] in configuration
+      - callback_plugins = ./callback_plugins
+      - clone git repo into <playbook root>/callback_plugins/ansible-telegraf-callback
+      - pip install pytelegraf (udp) or pytelegraf[http]
+      - running telegraf reachable by ansible controller configured with desired output to a metrics collector
+      - Tower http://docs.ansible.com/ansible-tower/latest/html/administration/tipsandtricks.html#using-callback-plugins-with-tower
+      - if running ansible < 2.4.0 you may need to remove the ini sections from this doc blob and use ENV vars
+    short_description: Sends play events to a telegraf forwarder (to metrics collector)
+    version_added: "2.4"
+    description:
+        - This is an ansible callback plugin that sends playbook stats to a telegraf listener input (udp/http) during playbook execution.
+        - Configure with <playbook root>callback_plugins/ansible-telegraf-callback/telegraf_callback.yml
+        - or using the module args or options in ansible.cfg
+    troubleshooting:
```

Comment:
this is not a valid documentation key, put in description or notes for now

Reply:
moved